libbeat/tests/system/requirements.txt

```diff
@@ -1,11 +1,29 @@
-pip
-nose
-jinja2
-PyYAML
-nose-timer
-redis
-elasticsearch
-requests
-autopep8
-six
-docker-compose
+autopep8==1.3.5
+backports.ssl-match-hostname==3.5.0.1
+cached-property==1.4.2
+certifi==2018.1.18
+chardet==3.0.4
+docker==3.2.1
+docker-compose==1.21.0
```

Comment:
What about updating also the version of docker-compose in `.travis.yml`?

Reply:
Done, in theory it should not affect the result, but :+1: to use the same one there